backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/comments/ViewRowComments.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * #%L
+ * metasfresh-webui-api
+ * %%
+ * Copyright (C) 2020 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.ui.web.comments;
+
+import com.google.common.collect.ImmutableMap;
+import de.metas.ui.web.window.datatypes.DocumentId;
+import lombok.Data;
+import lombok.NonNull;
+import lombok.Value;
+
+import java.util.Map;
+
+@Value
+@Data
+public class ViewRowComments
```

Comment:
pls decide if u want `@Value` or `@Data` :)

i would go with `@Value`

Reply:
Value is correct